server/models/User.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  username: { type: String },
+  email: { type: String },
+  password: { type: String },
+  dateCreated: { type: Date, default: new Date() },
+  dateEdited: { type: Date },
+  conversations: [{ type: Schema.Types.ObjectId, ref: "Conversation" }],
```

Comment:
For the same reason as the messages in conversations array, I rather we don't add this here, and index the users list in conversation schema

Reply:
Done and done!